### DISCUSSION TITLE
做个测试，看看图片能加载多长

### DISCUSSION THREAD

**Discussion by tayiie** [share121/inter-knot#3308] [其他]
![清明上河图 北宋 张择端绘 65500X1295像素 绢本设色 故宫博物院藏竖小](https://github.com/user-attachments/assets/dfc5c8d9-2d32-45f9-bb27-4fd6a9df478e)
## 看到封面图有的能显示长图，很好奇他最长能显示的有多长。

**Comment by Xia001275** [comment]
放出来就看到一条线awa
